### PR TITLE
Add VTEX IO cluster targeting

### PR DESCRIPTION
**Pull request by terciodemelo**
This PR intends to enable VTEX Toolbelt to target any VTEX IO cluster
by providing the routing cookie on some routes, and adding the
cluster id domain infix in other routes.

#### What is the purpose of this pull request?
To allow VTEX Toolbelt to communicate with any VTEX IO clutser
#### What problem is this solving?
This generalizes the communication between toolbelt and vtex io clusters, in such a way that we're no longe limited to two environments (prod and staging), but we can out-of-the-box to use it against any created environment

#### How should this be manually tested?
Basically execute common operations against multiple clusters, by specify the environment variable `VTEXIO_REGION`

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/env.ts [hunk 1]
```diff
@@ -25,3 +25,11 @@ export function region(): string {
 export function publicEndpoint(): string {
   return getEnvironment() === Environment.Staging ? 'myvtexdev.com' : 'myvtex.com'
 }
+
+export function clusterIdDomainInfix(): string {
```
Comment (igorframos) [inline src/env.ts]: What do you think of simply changing the `publicEndpoint()` function to return everything, including this region infix?

Comment (terciodemelo) [reply src/env.ts]: I thought about it, but most of the time we don't want the infix, because the cookie header will do it and axios lib won't handle the `set-cookie` by cloudfront. But when using `opn` you go to the browser and will get the cookie by going through the redirection process.

Now that you said it, I double checked and I should remove the infix from `sse.ts` and use the cookie header in `createEventSource`
